src/Analysis/GWASV2/Shared/ProgressBar/ProgressBar.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Steps } from 'antd';
+import { gwasV2Steps } from '../constants';
+import './ProgressBar.css';
+
+const { Step } = Steps;
+const ProgressBar = ({ current }) => (
+  <div className='progress-bar'>
+    <div className='progress-bar__steps'>
+      <Steps current={current}>
+        {gwasV2Steps.map((item, index) => (
+          <Step
+            key={item.title}
+            icon={<React.Fragment>{index + 1}</React.Fragment>}
+            title={`${current <= index ? item.title : item.secondaryTitle}`}
+          />
+        ))}
+      </Steps>
+    </div>
+    <Button>New to GWAS? Get started here</Button>
```

Comment:
this button will render on top of step4 if the screen is narrow enough
![Screenshot 2022-10-20 at 17 23 57](https://user-images.githubusercontent.com/2900303/196990960-34cacb1c-a091-446c-ac7c-40fdb8fd6747.png)

Reply:
Good catch! This should be fixed now.

Reply:
nice!